VideoLocker/res/values/strings.xml

```diff
@@ -296,6 +296,10 @@
     <string name="discussion_responses_endorsed">Endorsed</string>
     <!-- Label for time span when its less than a second after posting a response/comment -->
     <string name="just_now">Just now</string>
+    <!-- Label for letting the user know his/her response was successfully posted -->
+    <string name="discussion_response_posted">Your response is now posted</string>-->
+    <!-- Label for letting the user know his/her comment was successfully posted -->
+    <string name="discussion_comment_posted">Your comment is now posted</string>
```

Comment:
We are waiting on the docs team currently for the correct strings [here](https://openedx.atlassian.net/browse/MA-2162).